tests/TestRunner.Core/TestRunner.Core.NET.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk" >
+
+  <PropertyGroup>
+    <TargetFramework>net5.0-android</TargetFramework>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Xamarin.Android.UnitTests</RootNamespace>
+    <AssemblyName>TestRunner.Core.NET</AssemblyName>
```

Comment:
I'm curious can remove `$(AssemblyName)` here and it uses the name of the `.csproj` file?